### PR TITLE
Update free-programming-books-subjects.md

### PR DESCRIPTION
**Pull request by Imbiber**
## What does this PR do?
Added resource 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -223,6 +223,7 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 * [Computational Thinking](https://www.cs.cmu.edu/~15110-s13/Wing06-ct.pdf) - Jeannette Wing, Carnegie-Mellon University (PDF)
 * [Computer Science I](https://cse.unl.edu/~cbourke/ComputerScienceOne.pdf) - Chris Bourke (PDF)
 * [Computer Science II](https://cse.unl.edu/~cbourke/ComputerScienceTwo.pdf) - Chris Bourke (PDF)
+* [Computer_Science_Python_Book_Class_XI](https://cbseacademic.nic.in/web_material/doc/cs/1_Computer-Science-Python-Book-Class-XI.pdf) - CBSE (PDF)
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:226]: Don't invent titles! Use the title given by the resource.

Comment (Imbiber) [reply books/free-programming-books-subjects.md]: Good Morning sir,
I did the necessary changes, can you review it once.

**Comment by eshellman** [comment]
wrong list - this book is not english

**Comment by Imbiber** [comment]
> wrong list - this book is not english

Sorry sir , The book is entirely in English apart form two pages and since Hindi being the National Language of India, there are two pages one a beautiful poem and other contains a tiny glimpse of Our constitution whose translation can be found in page 6. It's government issued book, (probably the best book for any student studying CS in class 11) A very Vast population of student studies this during their school days. Those 2 pages signifies Hindi language as heritage of India.

**Comment by eshellman** [comment]
we'll accept it in the subjects list under 'Computer Science'

**Comment by Imbiber** [comment]
> we'll accept it in the subjects list under 'Computer Science'

Should I create a new PR for this?

**Comment by eshellman** [comment]
use same PR

**Comment by Imbiber** [comment]
> we'll accept it in the subjects list under 'Computer Science'

Sir I updated the book under the same only i.e. "Under Subjects -> Computer Science". Or I am wrong, Could you guide me, what should I do next.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by eshellman** [comment]
recently added

**Pull request merged**